src/gafaelfawr/config.py

```diff
@@ -168,41 +215,49 @@ class Config:
     issuer: IssuerConfig
     """Configuration for internally-issued tokens."""
 
+    verifier: VerifierConfig
+    """Configuration for the token verifier."""
+
     github: Optional[GitHubConfig]
     """Configuration for GitHub authentication."""
 
     oidc: Optional[OIDCConfig]
     """Configuration for OpenID Connect authentication."""
 
-    known_scopes: Dict[str, str]
+    known_scopes: Mapping[str, str]
     """Known scopes (the keys) and their descriptions (the values)."""
 
-    username_claim: str
-    """Token claim from which to take the username."""
-
-    uid_claim: str
-    """Token claim from which to take the UID."""
-
     safir_config: SafirConfig
```

Comment:
Rename as `safir` so config is not as redundant when accessing the attribute?

Reply:
Good idea. Will do.